core/App/screens/Connection.tsx

```diff
@@ -103,6 +105,10 @@ const Connection: React.FC<ConnectionProps> = ({ navigation, route }) => {
     if (notificationRecord) {
       switch (notificationRecord.type) {
         case 'CredentialRecord':
+          Toast.show({
```

Comment:
Are we changing the spec to do a toast here or is this supposed to be part of the screen?

Reply:
This is the current wireframe https://xd.adobe.com/view/3db881d4-e963-42ec-a2c4-6b5ca23862a3-68eb/screen/a523d262-2549-4da9-9c2e-35ba1a84d7e8/

Reply:
ah, I see. I was missing the wireframe, I'll refactor my changes

Reply:
@wadeking98 If that'll take a bit can you switch this to "Draft". Ty.